mne/viz/decoding.py

```diff
@@ -11,6 +11,127 @@
 import numpy as np
 import warnings
 
+def plot_topopatterns(csp, layout=None, vmin=None, vmax=None, sensors=True,
```

Comment:
I would just use the classic plot_topomap, and perhaps add a metrics patterns (as opposed to 'eeg', 'seeg', 'meg' etc)